mix.exs

```diff
@@ -40,7 +40,7 @@ defmodule Appsignal.Mixfile do
      compilers: compilers(Mix.env),
      elixirc_paths: elixirc_paths(Mix.env),
      deps: deps(),
-     docs: [logo: "logo.png"],
+     docs: [main: "Appsignal", logo: "logo.png"],
```

Comment:
does this make it point to the `Appsignal` module as the docs index page, rather than the api reference?

Reply:
Yes, that's it. Stole that idea from the [ExUnit docs](https://hexdocs.pm/ex_unit/ExUnit.html) to have a place to introduce the library and link to the install docs. The module list is still visible in the sidebar.